src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -438,7 +440,10 @@ class Message(pydantic.BaseModel):
     Single line content.
     """
 
-    ray_filename: RayFilename
+    # Pydantic v2 lets us do multiple aliases
+    # TODO: ORQSDK-956 - update Pydantic v2 and use AliasChoice
+    log_filename: Optional[LogFilename] = None
+    ray_filename: Optional[RayFilename] = None
```

Comment:
**issue**: `ray_filename` is a required property in the [API schema](https://github.com/zapatacomputing/workflow-driver/blob/17a53d42f42f1f90ee1c1c1eba2032117a5380e5/openapi/src/resources/workflow-run-logs.yaml#L28). It's a good practice to catch invalid data as soon as it enters our subsystem. Marking the property as optional would prevent us from identify the error.